### ISSUE TITLE
fix not working on eslint@8.3.0

### ISSUE THREAD

**Issue by ledzefflin** [peakchen90/eslint-plugin-react-directives#1]
Hi! 👋 
      
Firstly, thanks for your work on this project! 🙂

Today I used [patch-package](https://github.com/ds300/patch-package) to patch `eslint-plugin-react-directives@2.0.0` for the project I'm working on.

Here is the diff that solved my problem:

```diff
diff --git a/node_modules/eslint-plugin-react-directives/lib/rules/no-undef.js b/node_modules/eslint-plugin-react-directives/lib/rules/no-undef.js
index 28bd091..cdc99c2 100644
--- a/node_modules/eslint-plugin-react-directives/lib/rules/no-undef.js
+++ b/node_modules/eslint-plugin-react-directives/lib/rules/no-undef.js
@@ -1,5 +1,32 @@
-const baseRule = require('eslint/lib/rules/no-undef');
 const { getSettings } = require('../util');
+// const baseRule = require('eslint/lib/rules/no-undef');
+const baseRule = {
+  meta: {
+    type: "problem",
+
+    docs: {
+      description: "disallow the use of undeclared variables unless mentioned in `/*global */` comments",
+      recommended: true,
+      url: "https://eslint.org/docs/rules/no-undef"
+    },
+
+    schema: [
+      {
+        type: "object",
+        properties: {
+          typeof: {
+            type: "boolean",
+            default: false
+          }
+        },
+        additionalProperties: false
+      }
+    ],
+    messages: {
+      undef: "'{{name}}' is not defined."
+    }
+  }
+};
 
 /**
  * Checks if the given node is the argument of a typeof operator.
```

<em>This issue body was [partially generated by patch-package](https://github.com/ds300/patch-package/issues/296).</em>

**Comment by peakchen90** [comment]
Sorry I don't understand what you mean, can you provide more information and what is the problem

**Comment by ledzefflin** [comment]
@peakchen90 
First of all, I really appreciate your very convenient packages!!

This is the error message when I run "eslint --fix'' commend on app that uses eslint@8 or created by the latest "create-react-app". 
And above diff is to fix the issue.

I hope this information is helpful for your pacakge.

--
eslint --fix src --ext .js.jsx


Oops! Something went wrong! :(

ESLint: 8.9.0

Error [ERR_PACKAGE_PATH_NOT_EXPORTED]: Package subpath './lib/rules/no-undef' is not defined by "exports" in app/node_modules/eslint/package.json
    at new NodeError (internal/errors.js:322:7)
    at throwExportsNotFound (internal/modules/esm/resolve.js:332:9)
    at packageExportsResolve (internal/modules/esm/resolve.js:565:3)
    at resolveExports (internal/modules/cjs/loader.js:450:36)
    at Function.Module._findPath (internal/modules/cjs/loader.js:490:31)
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:888:27)
    at Function.Module._load (internal/modules/cjs/loader.js:746:27)
    at Module.require (internal/modules/cjs/loader.js:974:19)
    at require (/home/j/Git/cps-mweb/node_modules/v8-compile-cache/v8-compile-cache.js:159:20)
    at Object.<anonymous> (/home/j/Git/cps-mweb/node_modules/eslint-plugin-react-directives/lib/rules/no-undef.js:1:80)
npm ERR! code ELIFECYCLE
npm ERR! errno 2
npm ERR! @ lint:fix: `eslint --fix src --ext .js.jsx`
npm ERR! Exit status 2

**Comment by peakchen90** [comment]
@ledzefflin Thanks, I might know what is causing this problem, and I need some time to fix

**Comment by peakchen90** [comment]
@ledzefflin Fixed in v3.0.0
